source/expressions.tex

```diff
@@ -4049,9 +4049,8 @@
 An lvalue of type ``\cvqual{cv1} \tcode{B}'', where \tcode{B} is a class
 type, can be cast to type ``reference to \cvqual{cv2} \tcode{D}'', where
 \tcode{D} is a complete class derived\iref{class.derived} from \tcode{B},
-if \cvqual{cv2} is the
-same cv-qualification as, or greater cv-qualification than,
-\cvqual{cv1}. If \tcode{B} is a virtual base class of \tcode{D}
+if \cvqual{cv2}~\tcode{D} is at least as cv-qualified as \cvqual{cv1}~\tcode{B}.
```

Comment:
I am concerned that `const int` "is at least as cv-qualified" as `volatile int`, but not "the same or greater".

Reply:
Agreed.  We only have a partial order here.

Reply:
The PR now uses the wording "equally as or more cv-qualified than" (also fixes [expr.cond]), which should be unambiguous even with only a partial order.

Reply:
Progress, but I remain concerned that "equally as cv-qualified" does not mean "has the same cv-qualification" due to our partial ordering.